rdv_Dynamic shadows user stories

TuringsRevenge opened the pull request:
Let me know what you think. I can't think of anything else that a user would write.

TuringsRevenge committed:
Added Features and User Stories

TuringsRevenge committed:
Grammar

TuringsRevenge committed:
Updated Features.md

Syntax was not right. Didn't realize a space was needed before each.

swartzy4 reviewed (commented):
We could possibly have another feature that says something like, "As a Minecraft user I want the shadows to move based on the suns position in the sky for a more immersive experience." assuming that this is not already a feature already in the game.

swartzy4 reviewed (changes requested):
The sun will need to be added as well

TuringsRevenge committed:
Updated Features.md

Added features to add a sun and its movements and the objects responding to its change in position.

TuringsRevenge commented:
Updated with sun features.

swartzy4 reviewed (approved):
Seems good to me now
